pkg/registry/apis/datasource/plugincontext.go

```diff
@@ -20,14 +21,14 @@ import (
 // limit which namespace/tenant/org we are talking to
 type PluginDatasourceProvider interface {
 	// Get gets a specific datasource (that the user in context can see)
-	Get(ctx context.Context, pluginID, uid string) (*v0alpha1.DataSourceConnection, error)
+	Get(ctx context.Context, uid string) (*v0alpha1.DataSourceConnection, error)
```

Comment:
Is there a unique provider for each pluginID?  The point here is to get something that can be backed to k8s urls eventually, so we do not want "uid" to lookup both the pluginId and uid